src/main/scala/is/hail/methods/VEP.scala

```diff
@@ -308,74 +311,119 @@ object VEP {
         it
           .grouped(localBlockSize)
```

Comment:
Grouped won't work here since it collects values into an `ArrayBuffer`, but the underlying region values are changing.

Reply:
You might map to Variants and then group.